heron/schedulers/src/java/com/twitter/heron/scheduler/aurora/AuroraCLIController.java

```diff
@@ -111,29 +112,33 @@ public void removeContainers(Set<PackingPlan.ContainerPlan> containersToRemove)
   }
 
   @Override
-  public void addContainers(Integer count) {
+  public Set<Integer> addContainers(Integer count) {
     //aurora job add <cluster>/<role>/<env>/<name>/<instance_id> <count>
     //clone instance 0
     List<String> auroraCmd = new ArrayList<>(Arrays.asList(
-        "aurora", "job", "add", "--wait-until", "RUNNING", jobSpec + "/0", count.toString()));
-
-    if (isVerbose) {
-      auroraCmd.add("--verbose");
-    }
+        "aurora", "job", "add", "--wait-until", "RUNNING",
+        jobSpec + "/0", count.toString(), "--verbose"));
 
     LOG.info(String.format("Requesting %s new aurora containers %s", count, auroraCmd));
-    if (!runProcess(auroraCmd)) {
+    StringBuilder stderr = new StringBuilder();
+    if (!runProcess(auroraCmd, null, stderr)) {
       throw new RuntimeException("Failed to create " + count + " new aurora instances");
     }
+
+    String stdoutStr = stderr.toString();
+    String pattern = "Querying instance statuses: [";
```

Comment:
Not good to match against "Querying instance statuses: ". I think it is better to match against regex ".*\[(\S*)\]" and extract the string within [] and then parse it.

Reply:
`[]` is not enough, the stdout includes another [] list which is not container id list.